hedera-node/hedera-mono-service/src/main/java/com/hedera/services/contracts/execution/HederaMessageCallProcessor.java

```diff
@@ -51,30 +44,17 @@ public HederaMessageCallProcessor(
             final EVM evm,
             final PrecompileContractRegistry precompiles,
             final Map<String, PrecompiledContract> hederaPrecompileList) {
-        super(evm, precompiles);
+        super(evm, precompiles, hederaPrecompileList);
         hederaPrecompiles = new HashMap<>();
```

Comment:
nit: hederaPrecompiles map is contained and populated the same way in the parent class HederaEvmMessageCallProcessor so maybe it can be declared only there as protected field?